src/Generated/Models/Responses/CodeInterpreterCallOutput.Serialization.cs

```diff
@@ -44,8 +44,6 @@ protected virtual void JsonModelWriteCore(Utf8JsonWriter writer, ModelReaderWrit
                 writer.WritePropertyName("type"u8);
                 writer.WriteStringValue(Kind.ToString());
             }
-
-            Patch.WriteTo(writer);
```

Comment:
@jorgerangel-msft - This shouldn't be removed. Could you please take a look?

Reply:
Can we add a regression test that uses this model?